lib/pry/repl_file_loader.rb

```diff
@@ -21,12 +21,9 @@ def initialize(file_name)
 
     # Switch to interactive mode, i.e take input from the user
     # and use the regular print and exception handlers.
-    # @param [Pry] _pry_ the Pry instance to make interactive.
-    def interactive_mode(_pry_)
-      _pry_.config.input = Pry.config.input
-      _pry_.config.print = Pry.config.print
```

Comment:
This explicitly resets the values. With the new code nothing will be reset if `_pry_.config.input` and friends are set to something. Is this really the intention here? Maybe we don't need to change anything?